source/administration/config-settings.rst

```diff
@@ -1123,7 +1132,7 @@ The attribute in the AD/LDAP server used as a unique identifier in Mattermost. I
 
 If a user's ID Attribute changes, it will create a new Mattermost account unassociated with their old one.
 
-If you need to change this field after users have already logged in, use the `mattermost ldap idmigrate <https://about.mattermost.com/default-platform-ldap-idmigrate>`_ CLI tool.
+If you need to change this field after users have already logged in, use the `mattermost ldap idmigrate <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-ldap-idmigrate>`_ CLI tool.
```

Comment:
@wiersgallak I'd prefer to keep the .com redirect. That way if the redirect changes, we only have to change it once for https://about.mattermost.com/default-platform-ldap-idmigrate

Reply:
Actually, looks like the redirect did change :)

Have noted for marketing https://oss.mattermost.com/mattermost/pl/j9qq6oo47pyo8xjz8srucfmspr

Reply:
I changed the link as it wasn't working and I didn't realize it's a website link